docs/_includes/attr-element.adoc

```diff
@@ -4,7 +4,151 @@ Included in:
 - user-manual: Attributes: Setting attributes on an element
 ////
 
+
+////
+
+...there's still some confusion between document attributes and element attributes. We should make this distinction very clear.
+
+In general, Asciidoctor uses attributes as a way to store and deliver metadata. Document attributes are available everywhere and are used to control behavior of the document (they have a broad scope). Element attributes are specifically for controlling an individual element and are scoped to that element. Unlike document attributes, element attributes cannot be referenced using an attribute reference (though, that could change in UniDoc).
+
+Element here refers to a node or a directive. A node is a paragraph, a delimited block, a block macro, an inline macro or a span of formatted text. (maybe that needs to go in the glossary).
+////
+
+// tag::revised[]
+
+
+Element attributes are defined in "attribute lists" between square brackets, and apply only to the immediately following "element",
+where an "element" can be:
+
+* a paragraph
+* a delimited block
+* a block macro
+* an inline macro
+* a span of formatted text
+
+The syntax for setting element attributes is different from that for document attributes, and there is shorthand available for the most common attributes.
+
+For example, in:
+
+----
+[WARNING]
+Keep hands away from moving machinery
+----
+
+* The attribute list contains a single attribute `style` (a "positional attribute"), which has the value `WARNING`. 
+
+* The element that it applies to is the single paragraph "Keep hands away from moving machinery".
+
+Element attributes override document attributes, so that if the example had been preceded by 
+
+----
+:style: quote
+----
+
+The style of the paragraph would still have been WARNING, 
+and the value of the `style` attribute before and after the paragraph would still be `quote`.
+
+==== Element attribute syntax
+
+Element attributes can be set using:
+
+* named attributes
+
+* shorthand
+
+* positional attributes
```

Comment:
I think that we should explain that the most common positional attribute is the name of a block, such as "quote". However, the recognized positional attributes are determined exclusively by the block, block macro or inline macro. For instance, a source block has three positional attributes: block name, language, line numbers flag (technically line numbers should be an option, but that's a separate discussion). The image macros have three positional attributes: alt text, width, height. At some point, we should probably document all of these. For now, it's enough to point out that they exist.